wasm/event-stubs/my-component.c

```diff
@@ -0,0 +1,42 @@
+#include "hermes.h"
+
+// cspell: words cardano
```

Comment:
Better move this word into the `project.dic` file, because I suppose it would be widely used.